assets/docs/neso613/ASR_TFLite/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Instead of adding a license file directly, the license is specified in markdown (you have already done this). This file can be removed.

Reply:
ok. will do this.